src/v/coproc/tests/offset_storage_utils_tests.cc

```diff
@@ -117,38 +113,24 @@ FIXTURE_TEST(offset_keeper_saved_offsets, offset_keeper_fixture) {
 
     /// Attempt to retrieve the data that should have been written to disk
     /// We can assume this because the coproc_offset_flush_interval is
-    /// within the sleep interval
+    /// within the sleep interval (500ms set in the fixture parent class
+    /// constructor)
     ss::sleep(1s).get();
```

Comment:
That's a good idea. Now that I think about it, I think it might be good enough to wait until a `.coprocessor_stored_offsets` file is observed on disk.